lms/envs/common.py

```diff
@@ -3268,6 +3280,9 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 
     # Blockstore
     'blockstore.apps.bundles',
+
+    # MFE API
+    'lms.djangoapps.mfe_api',
```

Comment:
why don't you call it mfe instead of mfe_api? and use eg mfe/api/v1/views (out of curiosity)

Reply:
Because I did't want to create a big mfe app, but just a small API, and I saw that there are several apps that have the 'api' as a prefix or suffix like: lms.djangoapps.course_api, lms.djangoapps.mobile_api, etc.
If you have any ideas or comments to make it better, I would appreciate it.

Reply:
That's good! Thanks for the insight